antipatterns/maybe_it_is_not_rpc.json

```diff
@@ -17,7 +17,8 @@
         "@inproceedings{palma2015study, title={A study on the taxonomy of service antipatterns}, author={Palma, Francis and Mohay, Naouel}, booktitle={Patterns Promotion and Anti-patterns Prevention (PPAP), 2015 IEEE 2nd Workshop on}, pages={5--8}, year={2015}, doi={10.1109/PPAP.2015.7076848}}"
       ],
       "tags": [
-        "antipattern",
+        "soa",
+        "architecture",
```

Comment:
Maybe we should add the tag `rpc`. There are multiple antipatterns mentioning rpc as cause or solution.
E.g. Maybe It's Not RPC, CRUDy Interface,.